experimental/dds/tree/src/EditLog.ts

```diff
@@ -242,12 +269,14 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 	 * Construct an `EditLog` using the given options.
 	 * @param summary - An edit log summary used to populate the edit log.
 	 * @param logger - An optional logger to record telemetry/errors
+     * @param numberOfAvailableSequencedEdits - The number of the latest acked edits that will be available.
 	 */
 	public constructor(
 		summary: EditLogSummary<TChange, EditHandle<TChange>> = { editIds: [], editChunks: [] },
 		logger?: ITelemetryLogger,
 		editAddedHandlers: readonly EditAddedHandler<TChange>[] = [],
-		indexOfFirstEditInSession = summary.editIds.length
+		indexOfFirstEditInSession = summary.editIds.length,
+        numberOfAvailableSequencedEdits = Number.MAX_SAFE_INTEGER
```

Comment:
I think I prefer infinity here. It conveys the intent to keep _all_ of the edits, not just up to a finite (though enormous) limit. It does make the initialization logic slightly more annoying, though :)